src/include/partition_filter.h

```diff
@@ -43,17 +43,15 @@ typedef struct
 } ResultRelInfoHolder;
 
 
-/* Forward declaration (for on_new_rri_holder()) */
+/* Forward declaration (for on_rri_holder()) */
```

Comment:
Is this a necessary change? This callback is indended to be called when a **new** ResultRelInfo holder is created.